### ISSUE TITLE
3D classification does not run eventhough it appears in running jobs without errors

### ISSUE THREAD

**Issue by baratachencho27** [3dem/relion#1110]
Hi you all, I have a very strange scenario. I am using relion 5.0 and I am running some 3D classifications. However, yesterday I launched a  Class3D job on the GUI and the job started without any error, so I let it and and left.
This morning I saw that the job was still in the running jobs window but nothing is written on neither the run.out nor run.error.

When I check the processes running on my computer, I see that the process is running: 

serg   2690205 99.8  0.0  62900 12448 pts/3    R+   13:57  23:18 mpirun -n 5 /opt/relion/build/bin/relion_refine_mpi --o Class3D/job468/run --i Select/job456/particles.star --ref Select/job449/newvolume.mrc --firstiter_cc --ini_high 50 --dont_combine_weights_via_disc --pool 30 --pad 2 --ctf --iter 25 --tau2_fudge 4 --particle_diameter 1200 --K 4 --flatten_solvent --zero_mask --oversampling 1 --healpix_order 2 --offset_range 10 --offset_step 2 --sym C1 --norm --scale --helix --helical_inner_diameter 500 --helical_outer_diameter 790 --helical_nr_asu 2 --helical_twist_initial 12.24 --helical_rise_initial 5.5 --helical_z_percentage 0.3 --helical_symmetry_search --helical_twist_min 11.02 --helical_twist_max 13.46 --helical_rise_min 4.95 --helical_rise_max 6.05 --helical_keep_tilt_prior_fixed --sigma_tilt 5 --sigma_psi 3.33333 --sigma_rot 0 --j 16 --pipeline_control Class3D/job468/

I still hard disk memory in /home

Thanks in advance

**Comment by biochem-fan** [comment]
Does this always occur? Or did this happen only once? In the latter case, I wouldn't worry.

**Comment by baratachencho27** [comment]
It didn´t happen till now. From now on, it´s always happening. Therefore, I can´t run it.

**Comment by biochem-fan** [comment]
If you re-run earlier successful jobs, do they run?

**Comment by baratachencho27** [comment]
They don´t

**Comment by biochem-fan** [comment]
If you didn't recompile RELION right before the problem started, this is probably your system problem, not RELION's. Did you reboot the machine? Didn't you install something (e.g. conda)? Didn't the system update packages?

**Comment by baratachencho27** [comment]
As far as I know B No. Might it be related to MPI? because for some reason, I have ran one 3D class with 1 MPI (meaning no parallel nodes used) and it´s working now...

**Comment by biochem-fan** [comment]
Make sure you are using the same version of MPI runtime as the one used to compile RELION.

**Comment by baratachencho27** [comment]
It shouldn´t have changed, if it worked two days ago, and not now. Isn´t is?

**Comment by biochem-fan** [comment]
Something must have changed unintentionally. Because you didn't recompile RELION, we can be sure that RELION hasn't changed. Thus, this is not RELION's problem and I cannot help.

**Comment by baratachencho27** [comment]
You are right, Thanks for the piece of advice. I am updating to the newest version of open.mpi. Should I recompile relion afterward?
If I recompile relion, I do not lose my workflows and results right?

**Comment by biochem-fan** [comment]
> Should I recompile relion afterward?

Yes.

> If I recompile relion, I do not lose my workflows and results right?

Your results are safe.

**Comment by baratachencho27** [comment]
Hi, I am wondering If the latest version of MPI open (5.0.3-1) is compatible witht Relion.

Thank u

**Comment by biochem-fan** [comment]
It should.

**Comment by xiaodiyu** [comment]
Hi baratachencho27, did you resolve your issue with recompiling the relion 5 with MPI (5.0.3)? I had similar issue that relion will use the GPUs for several mins and then stuck there showing 0% of GPU-Util wihout any error message.

![Capture](https://github.com/3dem/relion/assets/103237465/d0e609dc-42dc-4227-98bd-eb883934b75d)

**Comment by baratachencho27** [comment]
Hi xiaodiyu,
Yes I did. Apparently it was kind of a bug of my machine with the OpenMPI. I upgraded the OpenMPI to (5.0.3) and it wasn´t working at all. Then I reboot the system and suddenly the mpirun program started to work.
I download and recompile RELION 5 again because I modify the varible paths to OpenMPI and now it´s wprking well.
Have to tried to reboot?

Best

**Comment by xiaodiyu** [comment]
Hi [baratachencho27,

we were able to recompile relion 5 with correct path. right now it is running great.

Thanks,

Xiaodi

**Comment by biochem-fan** [comment]
Now that both people solved the problem, I close this issue.

**Comment by baratachencho27** [comment]
Hi all, yeah apparently it not solved yet. Eventhoug the 3D Classification runs properly. I have tried now the 3D refinement and it tasks for ever. It reached the itineration 19 but nothing else is happening.
I looked at the top and nvidia-smi, and even though most 100% GPU is taken,about 0-0.4% RAM is used.
Also I got no running processes as you had.
![image](https://github.com/3dem/relion/assets/166137076/401ec7ec-2a15-4481-a314-3756e7850ca4)
